libraries/ESP8266WiFi/src/ESP8266WiFiGeneric.cpp

```diff
@@ -451,9 +451,9 @@ bool ESP8266WiFiGenericClass::mode(WiFiMode_t m) {
     //tasks to wait correctly.
     constexpr unsigned int timeoutValue = 1000; //1 second
     if(can_yield()) {
-        // The final argument, intvl_ms, to esp_delay influences how frequently
-        // the scheduled recurrent functions (Schedule.h) are probed.
-        esp_delay(timeoutValue, [m]() { return wifi_get_opmode() != m; }, 5);
+        // The final argument, intvl_ms, to esp_delay determines at least
+        // how frequently wifi_get_opmode() is checked
```

Comment:
> 'determines at least'

...does not sound right :)
minimal delay? interval between callback invocations, which decide whether to continue blocking?

-//- for the rest of comments, we repeat either almost the same thing or state what esp\_delay documentation already says